src/Schema/Tables/Appointments/Appointments.sql

```diff
@@ -0,0 +1,16 @@
+﻿CREATE TABLE [dbo].[Appointments] 
+  ( 
+     [AppointmentId]   INT NOT NULL IDENTITY(1, 1), 
+     [PatientId]       INT NOT NULL, 
+     [UserId]          INT NOT NULL, 
+     [AppointmentDate] DATE NOT NULL DEFAULT Getdate(), 
+     [AppointmentTime] TIME NOT NULL, 
```

Comment:
@DenyaAbramenkov  Why have you decided to split the date into two columns?

Reply:
It seems to working properly, and I think it will be more convenient for user to search. But it can be changed.